frontend/amer-fort-explorer/index.html

```
@@ -0,0 +1,533 @@
+          <li><strong>Department of Archaeology &amp; Museums, Government of Rajasthan:</strong> <em>Guide to Amber Fort and Palaces</em>, Jaipur.</li>
+        </ul>
+      </div>
+    </section>
+  </main>
+
+  <!-- Lightbox Modal -->
+  <div class="amer-modal" id="amer-modal" aria-hidden="true" role="dialog" aria-label="Image Lightbox">
+    <div class="amer-modal-overlay" id="amer-modal-overlay"></div>
+    <div class="amer-modal-dialog">
+      <button class="amer-modal-close" id="amer-modal-close" aria-label="Close Lightbox">&times;</button>
+      <img id="amer-modal-img" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)